perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/beans/Member.java

```diff
@@ -33,6 +34,31 @@ public static final Member createNew(int id, int voId, int userId, String member
 		member.setMembershipType(membershipType);
 		member.setMembershipStatus(membershipStatus);
 		member.setObjectType("Member");
+		member.setSponsored(false);
```

Comment:
To avoid duplicit code, you should call here the other constructor with `false` value for sponsored field.